docx/section.py

```diff
@@ -37,11 +38,25 @@ class Section(object):
     """
     Document section, providing access to section and page setup settings.
     """
+    _header = None
+
     def __init__(self, sectPr):
         super(Section, self).__init__()
         self._sectPr = sectPr
 
     @property
+    def header(self):
+        if not self._header:
+            # here's how I did the header element in my previous implementation:
+            # from .oxml.header import CT_Hdr
+            # header_elm = CT_Hdr.new()
+            # not sure what is appropriate here for this first unit test
+            header_elm = 'pretend element'
+            self._header = Header(header_elm, self)
+
+        return self._header
```

Comment:
used a pretend element here because i wasn't sure how to get a real element without writing a bunch more tests of the new CT_Hdr element type.